packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -170,9 +170,7 @@ class DebuggerController extends DisposableController
     // and modify to detect if app is paused from the isolate
     // https://github.com/flutter/devtools/pull/4993#discussion_r1060845351
 
-    serviceManager.appState
-      ..setIsolateRef(ref)
-      ..setPausedOnBreakpoint(false);
+    serviceManager.appState..setPausedOnBreakpoint(false);
```

Comment:
remove the .. now that there is only one method call

Reply:
Filed https://github.com/dart-lang/linter/issues/3960 to consider adding a lint to handle to warn about this case.